source/client-side-encryption/etc/test-templates/fle2v2-Rangev2-Defaults.yml.template

```diff
@@ -0,0 +1,157 @@
+                          "subType": "06"
+                      }
+                  }
+              }
+            encryptionInformation: *encryptionInformation
+          command_name: find
+    outcome:
+      collection:
+        data:
+        -
+          {
+            "_id": 0,
+            "encryptedInt": { $$type: "binData" },
+            # Expected contents of `__safeContent__` require MONGOCRYPT-698 to apply expected `trimFactor`.
```

Comment:
Confirming: is this test deterministic? Prose test notes that payload contents should not be compared due to random data.

Reply:
Yes. The find payloads for automatic encryption are deterministic.

Insert payloads depend on random data. This is why the expectation for the "insert" command only asserts `"encryptedInt": { $$type: "binData" }`.

The prose test notably uses an insert payload rather than a find payload. For explicit encryption, find payloads include a [process-unique payloadId](https://github.com/mongodb/libmongocrypt/blob/e81ef62825c8034645f107f4026cf874b60285a4/src/mc-fle2-rfds-private.h#L89-L91) which prevent exact comparison.

However, the spec test does exact comparison for find payloads. Auto encryption use deterministic payloadId's provided by query analysis.